extensions/amp-bind/0.1/storybook/Basic.amp.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {boolean, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+import {withAmp} from '@ampproject/storybook-addon';
+
+export default {
+  title: 'amp-bind',
```

Comment:
Imho this should live in the `amp-accordion` extension and we can just call the file `Bind.amp.js` and the title would be `amp-accordion with amp-bind` or something like this. I.e. we want all extensions that have interesting behavior have their own binder stories instead of overwhelming the amp-bind extension.

Reply:
Thanks for the feedback Dima, I've moved the story to `Bind.amp.js` and tried changing its name.  However, when testing locally, the storybook does not pick up this new story.  I had also tried changing the `_default` to some other value, but still it was not picked up.

Also, I have noticed a new problem now that I am receiving the following error each time I click on one of the buttons in the storybook and it no longer expands or collapses any of the accordion sections.  Do you have any ideas?  This error seems similar to the error we received with Integration tests.

![image](https://user-images.githubusercontent.com/25626770/97885978-0e288080-1cf6-11eb-9f61-cf900b84ff2e.png)

Reply:
This was resolved per our offline conversation, thanks!